examples/consensus_pub_sub/main.go

```diff
@@ -14,14 +14,11 @@ func main() {
 	var client *hedera.Client
 	var err error
 
-	if os.Getenv("HEDERA_NETWORK") == "previewnet" {
-		client = hedera.ClientForPreviewnet()
-	} else {
-		client, err = hedera.ClientFromConfigFile(os.Getenv("CONFIG_FILE"))
-
-		if err != nil {
-			client = hedera.ClientForTestnet()
-		}
+	net := os.Getenv("HEDERA_NETWORK")
+	client, err = hedera.ClientForName(net)
+	if err != nil {
+		println(err.Error(), ": error creating client")
+		return
 	}
 
 	configOperatorID := os.Getenv("OPERATOR_ID")
```

Comment:
Remove these variables and the conditional 
```go
	if configOperatorID != "" && configOperatorKey != "" && client.GetOperatorPublicKey().Bytes() == nil {
```
Instead do
```go
		operatorAccountID, err := hedera.AccountIDFromString(os.Getenv("OPERATOR_ID"))
		if err != nil {
			println(err.Error(), ": error converting string to AccountID")
			return
		}
```
Do the same for `OPERATOR_KEY`

Reply:
i see even more cleaner